### PR TITLE
Branch update documentation

### PR DESCRIPTION
**Pull request by prawnzyy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -207,6 +207,11 @@ steps start
 complete recipe
 ```
 
+* Name of Recipe needs to be of Alphanumeric format
+* Ingredients inputted **must** be of the format `(name) (quantity)(unit)` where an example is `flour 100g`
```
Comment (ItsTYtan) [inline docs/UserGuide.md:211]: is there a space in the brackets from quantity to unit?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -605,37 +677,49 @@ testers are expected to do more *exploratory* testing.
...
        Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
-
 ### Saving data
 
 1. Dealing with missing/corrupted data files
 
-    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+    1. Head over to the folder where your jar file is located and then delete both inventory.json and recipeBook.json within the data folder.
```
Comment (ItsTYtan) [inline docs/DeveloperGuide.md:684]: Small suggestion use "open" instead of "head over to", remove the "then" after the "and"

**Review by ItsTYtan — COMMENTED** [review]
Small suggestions, other than that need to see the actual website later to confirm the correct formatting.

**Review by TohLiYuan — APPROVED** [review]
lgtm!

**Pull request merged**